nautobot/extras/views.py

```diff
@@ -1984,42 +1984,32 @@ def destroy(self, request, *args, **kwargs):
         return super().destroy(request, *args, **kwargs)
 
 
-class ScheduledJobListView(generic.ObjectListView):
+class ScheduledJobUIViewSet(
```

Comment:
Can you mark up which is missing as a comment and why it's not there. Makes it easier to understand down the road.

Reply:
At a glance I *think* the only missing one is `ObjectCreateViewMixin`, which makes sense, but agreed, comments would be helpful.